### ISSUE TITLE
Corrupt tarball for mypy-boto3-ec2 on PyPI

### ISSUE THREAD

**Issue by hyandell** [youtype/mypy_boto3_builder#219]
**Describe the bug**
The tarball at https://pypi.org/project/mypy-boto3-ec2/#files is damaged - it doesn't untar. (tar xf)

**To Reproduce**
Download https://files.pythonhosted.org/packages/40/9a/13d6147d4b902195b4457788869a545041aeeed0c2c4527b0dbe0e03d150/mypy-boto3-ec2-1.28.29.tar.gz

Try to untar:

```
tar xf mypy-boto3-ec2-1.28.29.tar.gz
```

**Actual output**
```
tar: Damaged tar archive
```

**Expected output**

Successful unpacking of the package

**Additional context**
MacOS, Linux

**Comment by vemel** [comment]
Thank you for the report! I am going to rebuild it and reupload

**Comment by vemel** [comment]
I published `mypy-boto3-ec2 1.28.29.post1`.

Also, there is a consistency check for output `.whl` and `.tar.gz` files, so we should be safe from now. I wonder how many times it happened...

Please update and let me know if it works as it should for you.
